webapp/package-lock.json

```
@@ -4497,9 +4390,11 @@
       }
     },
     "mattermost-redux": {
-      "version": "github:mattermost/mattermost-redux#b56e7e3a6f7f5de281fd3541eefec9345a8bbc89",
-      "from": "github:mattermost/mattermost-redux#b56e7e3a6f7f5de281fd3541eefec9345a8bbc89",
+      "version": "5.23.0",
```

Running `npm install` locally, I got this change in `package-lock.json`. @larkox do you mind running it again to have it match what's in `package.json`?

```suggestion
      "version": "5.20.0",
```